monte_carlo.py

```diff
@@ -214,7 +225,7 @@ def transport_neutron(materials, bounds, tallies, fission_banks, first_round,
  @param     num_batches the number of batches to be tested
 '''
 def generate_neutron_histories(n_histories=0, materials=0, bounds=0,
-        mesh=0, num_batches=0):
+        mesh=0, num_batches=0, chi = [0,0,0]):
```

Comment:
Let's get rid of all default values here. It doesn't make too much sense to allow ``generate_neutron_histories`` to run without specifying the number of histories, the materials, the boundaries, mesh, or num batches. Also ``chi`` should be an attribute included in ``materials``.